lib/Jsrt/Core/JsrtContextCore.cpp

```diff
@@ -120,6 +120,36 @@ HRESULT ChakraCoreHostScriptContext::FetchImportedModule(Js::ModuleRecordBase* r
     return E_INVALIDARG;
 }
 
+HRESULT ChakraCoreHostScriptContext::FetchImportedModuleFromScript(JsSourceContext dwReferencingSourceContext, LPCOLESTR specifier, Js::ModuleRecordBase** dependentModuleRecord)
+{
+    if (fetchImportedModuleFromScriptCallback == nullptr)
+    {
+        return E_INVALIDARG;
+    }
+
+    HRESULT hr = NOERROR;
+    Js::JavascriptString* specifierVar = nullptr;
+    specifierVar = Js::JavascriptString::NewCopySz(specifier, GetScriptContext());
+    
+    if (FAILED(hr))
+    {
+        return hr;
+    }
+
+    JsModuleRecord dependentRecord = JS_INVALID_REFERENCE;
```

Comment:
there is not much difference between this function and above function (FetchImportedModule). Did you consider creating a lambda and just stub relevant part in that?